ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -14,30 +14,32 @@
     - '{{ docker_image_prefix }}/javaaction'
   when: docker_registry != ""
 
-- name: (re)start invoker
-  docker_container:
-    name: "invoker{{play_hosts.index(inventory_hostname)}}"
-    image: "{{docker_registry}}{{ docker_image_prefix }}/invoker:{{docker_image_tag}}"
-    state: started
-    recreate: true
-    restart_policy: "{{docker.restart.policy}}"
-    hostname: "invoker{{play_hosts.index(inventory_hostname)}}"
-    env:
-      "COMPONENT_NAME": "invoker{{play_hosts.index(inventory_hostname)}}"
-      "CONSULSERVER_HOST": "{{groups['consul_servers'] | first}}"
-      "CONSUL_HOST_PORT4": "{{consul.port.http}}"
-      "SELF_DOCKER_ENDPOINT": "{{inventory_hostname}}:{{docker.port}}"
-      "SERVICE_NAME": "invoker{{play_hosts.index(inventory_hostname)}}"
-      "SERVICE_CHECK_HTTP": "/ping"
-      "SERVICE_CHECK_TIMEOUT": "2s"
-      "SERVICE_CHECK_INTERVAL": "15s"
-      "PORT": 8080 
-    volumes:
-      - "{{whisk_logs_dir}}/invoker{{play_hosts.index(inventory_hostname)}}:/logs"
-      - "/var/lib/docker/containers/:/containers"
-    ports:
-      - "{{invoker.port + play_hosts.index(inventory_hostname)}}:8080"
-    command: /bin/sh -c "/invoker/bin/invoker {{play_hosts.index(inventory_hostname)}} >> /logs/invoker{{play_hosts.index(inventory_hostname)}}_logs.log 2>&1"
+- name: determine docker root dir
+  shell: "docker info 2>/dev/null | grep 'Docker Root Dir' | awk -F ': ' '{print $2}'"
+  register: docker_root_dir
```

Comment:
good idea. we can omit the version tho to get the latest. we have already pinned the api version implicitly by pinning the docker version.

```
curl host:port/info
```
should yield the same results. (calls /1.24/info on docker 1.12)